projects/storefrontstyles/scss/cxbase/_variables-bootstrap.scss

```diff
@@ -994,13 +994,6 @@ $spinner-width-sm: 1rem !default;
 $spinner-height-sm: $spinner-width-sm !default;
 $spinner-border-width-sm: 0.2em !default;
 
-// Close
```

Comment:
we should still use those vars. maybe move them somewhere else than just removing them